scripts/bfb-install

```diff
@@ -371,18 +371,51 @@ push_boot_stream()
   fi
 }
 
+check_nic_mode()
+{
+  local str
+
+  [ "$mode" != "local" ] && return
+
+  # Get PCIE BDF
+  str=`cat ${rshim_node}/misc | grep DEV_NAME | awk '{print $2}'`
+  str=${str:5}
+  str=${str/.*/.0}
+  pcie_bdf="$str"
+
+  # Check NIC mode
+  str=`mlxconfig -d "$str" q INTERNAL_CPU_OFFLOAD_ENGINE 2>/dev/null | grep INTERNAL_CPU_OFFLOAD_ENGINE | grep 'DISABLED(1)'`
```

Comment:
str=`mlxconfig -d "$str" -e q INTERNAL_CPU_OFFLOAD_ENGINE 2>/dev/null | grep 'DISABLED(1)'`

- This will query specific parameter and make it faster.

Reply:
Done